post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -241,6 +243,10 @@ protected void processCachedIds() {
                         processTopic(keyTopic, entity, ids,
                                 postProcRepository::executeStoredProcForNotificationIds);
                         break;
+                    case CASE_MANAGEMENT:
```

Comment:
Does this listen for a nrt_case_management event?

Reply:
Yes, it's processing phc uids from `nrt_investigation_case_management` topic